### PR TITLE
Flesh out the template README

### PR DESCRIPTION
**Pull request by parkerl**
@kytrinyx I pulled over the generic parts from the elixir and elm READMEs. What do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,57 @@
 # xSLUG
+![build status](https://travis-ci.org/exercism/x-api.svg?branch=master)
```
Comment (kytrinyx) [inline README.md]: Can we make this `xSLUG` instead of x-api?

File: README.md [hunk 2]
```diff
@@ -1,18 +1,57 @@
...
+## Setup
 
-_Document how to contribute to the LANGUAGE track._
+The simplest way to install SLUG is ...
 
-## Contributing Guide
+## Contributing
 
-Please see the [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data)
+Thank you so much for contributing! :tada:
```
Comment (kytrinyx) [inline README.md]: Yeah, much much better!

**Comment by kytrinyx** [comment]
So much better!

File: README.md (hunk 1 above)
Comment (parkerl) [inline README.md]: For the build status image? Yeah I was going to ask about that. I was trying to have it display a "real" status image. `xslug` gives `{"file":"not found"}` but we could use `x-template` which gives

Comment (kytrinyx) [inline README.md]: Yeah, that will mean we can't just use sed or whatever, but that seems fine. At least the build image won't be passing (which could be confusing).

**Comment by kytrinyx** [comment]
@parkerl Did we come to a conclusion on this one?

**Comment by parkerl** [comment]
@kytrinyx uhhh....what was I supposed to be doing on this one? 😕

**Comment by kytrinyx** [comment]
I can't remember. Rereading... the build image thingy, using xSLUG instead of x-api?

**Comment by parkerl** [comment]
Well this has been hanging here for a long time. Guess I should take a look.

**Comment by kytrinyx** [comment]
@parkerl I just saw it the other day when I renamed the repo, and was going to figure it out—but if you figure it out before I come out of my drugged post-surgical daze, that would be stellar.

**Comment by parkerl** [comment]
@kytrinyx how about now? Updated to have `{{LANGUAGE}}` instead of slug everywhere. Is that what you would like ti to be now?

File: TRACK_README.md [hunk 3]
```diff
@@ -1,11 +1,58 @@
...
-## TODO
+## Setup
+
+The simplest way to install {{LANGUAGE}} is ...
+
+## Contributing
+
+Thank you so much for contributing! :tada:
+
+Please start by reading the general Exercism [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data).
```
Comment (kytrinyx) [inline TRACK_README.md]: We've got a bit of a mess with respect to contributing guides. That one is now just about the problems API.

Should we link to https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md (It's a bit of a wall of text).

Or perhaps [getting involved in a track](https://github.com/exercism/exercism.io/blob/master/docs/getting-involved-in-a-track.md)?

File: TRACK_README.md [hunk 4]
```diff
@@ -1,11 +1,58 @@
...
+
+We welcome pull requests that provide fixes and improvements to existing exercises. If you're unsure, then go ahead and open a GitHub issue, and we'll discuss the change.
+
+Please keep the following in mind:
+
+- Pull requests should be focused on a single exercise, issue, or change.
+
+- We welcome changes to code style, and wording. Please open a separate PR for these changes if possible.
+
+- Please open an issue before creating a PR that makes significant (breaking) changes to an existing exercise or makes changes across many exercises. It is best to discuss these changes before doing the work.
```
Comment (kytrinyx) [inline TRACK_README.md]: This is great. We may want to point to the [exercism/discussions](https://github.com/exercism/discussions/issues) repo for broader discussions about an exercise.

File: TRACK_README.md [hunk 5]
```diff
@@ -1,11 +1,58 @@
...
+- Follow the coding standards for {{LANGUAGE}}. (If there is a formatter for the track's language, add instructions for using it here.)
+
+- Watch out for trailing spaces, extra blank lines, and spaces in blank lines.
+
+- Each exercise must stand on its own. Do not reference files outside the exercise directory. They will not be included when the user fetches the exercise.
+
+- Exercises should use only the {{LANGUAGE}} core libraries.
+
+- Please do not add a README or README.md file to the exercise directory. The READMEs are constructed using shared metadata, which lives in the
+[exercism/x-common](https://github.com/exercism/x-common) repository.
```
Comment (kytrinyx) [inline TRACK_README.md]: We can link to https://github.com/exercism/exercism.io/blob/master/docs/fixing-exercise-readmes.md here

File: TRACK_README.md [hunk 6]
```diff
@@ -1,11 +1,58 @@
...
+
+- Please do not commit any configuration files or directories inside the exercise other than ...
+
+- Test files should use the following format:
+
+```
+# include the body of an example test
+```
+
+ - All the tests for x{{LANGUAGE}} exercises can be run from the top level of the repo with ... Please run this command before submitting your PR.
```
Comment (kytrinyx) [inline TRACK_README.md]: This should be `x{{SLUG}}` or `{{LANGUAGE}}`.

File: TRACK_README.md [hunk 7]
```diff
@@ -1,11 +1,58 @@
...
+
+ - If you are submitting a new exercise, be sure to add it to the appropriate place in the `config.json` file. Also, please run `bin/fetch-configlet && bin/configlet` to ensure the exercise is configured correctly.
 
-_Document how to contribute to the {{LANGUAGE}} track._
+## License
 
-## Contributing Guide
+The MIT License (MIT)
```
Comment (kytrinyx) [inline TRACK_README.md]: Since we have a license file, and GitHub now displays the license on the repo page, we can probably ditch this section.

Comment (kytrinyx) [inline TRACK_README.md]: ... including the copyright (that is also in the license file).

**Comment by kytrinyx** [comment]
Yes, this is excellent. I have a few minor comments.

File: TRACK_README.md (hunk 3 above)
Comment (parkerl) [inline TRACK_README.md]: Both???

File: TRACK_README.md (hunk 4 above)
Comment (parkerl) [inline TRACK_README.md]: Will do

File: TRACK_README.md (hunk 5 above)
Comment (parkerl) [inline TRACK_README.md]: Excellent

File: TRACK_README.md (hunk 3 above)
Comment (kytrinyx) [inline TRACK_README.md]: Haha, yes. Both! "getting involved" first, and then the CONTRIBUTING after, perhaps saying that it has a bunch of topics, so scan the table of contents for things relevant to what you're trying to do.

File: TRACK_README.md [hunk 8]
```diff
@@ -1,11 +1,52 @@
...
-_Document how to contribute to the {{LANGUAGE}} track._
+```
+# include the body of an example test
+```
 
-## Contributing Guide
+ - All the tests for {{LANGUAGE}} exercises can be run from the top level of the repo with ... Please run this command before submitting your PR.
 
-Please see the [contributing guide](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md).
+ - If you are submitting a new exercise, be sure to add it to the appropriate place in the `config.json` file. Also, please run `bin/fetch-configlet && bin/configlet` to ensure the exercise is configured correctly.
```
Comment (kytrinyx) [inline TRACK_README.md]: Yes, excellent idea!

**Comment by parkerl** [comment]
What do you think? Ready?

**Comment by kytrinyx** [comment]
Ready!

Thanks @parkerl! :sparkles:

File: TRACK_README.md [hunk 9]
```diff
@@ -1,11 +1,52 @@
...
-## TODO
+## Setup
+
+The simplest way to install {{LANGUAGE}} is ...
+
+## Contributing
+
+Thank you so much for contributing! :tada:
+
+Please start by reading how to [get involved in a track](https://github.com/exercism/exercism.io/blob/master/docs/getting-involved-in-a-track.md). Comprehensive details on contributing can be found [here](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md). Be sure to read the Exercism [Code of Conduct](https://github.com/exercism/exercism.io/blob/master/CODE_OF_CONDUCT.md).
```
Comment (petertseng) [inline TRACK_README.md]: mark

**Pull request merged**